template/example_test.go

```diff
@@ -113,3 +107,51 @@ func ExampleTemplate_complexMessage() {
 
 	// Output: John is 42 years old and his favorite color is 255,0,0.
 }
+
+func ExampleTemplate_calculator() {
```

Comment:
Partly to show that the raw value has to be exposed. But also it's cool imho ;)

Reply:
Can you think of a better example?

The functions in the registry SHOULD be related to the localisation. I don't want to encourage developers to register functions that should be handled in Golang instead. Otherwise, it adds noise to the text and makes it harder for translators to maintain translations.

E.g. _:parity_ that returns the localised name for _odd_ or _even_.

The _:color_ example can be improved too.

Reply:
Will add TODO and address it in another PR.

Reply:
Keep it simple, delete it. Less noise.